src/Sarif.Viewer.VisualStudio/Themes/Information.xaml

```diff
@@ -3,12 +3,15 @@
                     xmlns:local="clr-namespace:Microsoft.Sarif.Viewer.Themes"
                     xmlns:converters="clr-namespace:Microsoft.Sarif.Viewer.Converters"
                     xmlns:controls="clr-namespace:Microsoft.Sarif.Viewer.Controls">
+    <ResourceDictionary.MergedDictionaries>
```

Comment:
nitpick: Can you add an empty line between the control declaration and the ResourceDictionary. I think it will make it easier to read.